vtex-apps/reviews-and-ratings
# [REVIEWS-91] Create a custom eevnt to update RatingsSummary component

#### What problem is this solving?

The rating summary component was not updating when a new review was saved in the front.
I create a new file inside the utils folder to export the eventBus object that contains three functions, i'm using the dispatch function to create a custom event, and the rest of the function could help for future implementations related with event listeners.

#### How to test it?
Go the workspace to test and create a new review, you should see the new review added in the front as well as the rating-summary block:
![Screen Shot 2022-05-12 at 14 30 15](https://user-images.githubusercontent.com/49737670/168153715-57afc470-3fd7-49de-8e81-24e89f9a774d.png)

[Workspace to test](https://arturoreviews--sandboxusdev.myvtex.com/invicta-pro-diver-swiss-movement-quartz-watch---stainless-steel-case---model-0003/p)
